LearningHub.Nhs.WebUI/Styles/nhsuk/pages/mylearning.scss

```diff
@@ -2,7 +2,7 @@
 
 
 .my-learning {
-   
+
```

Comment:
are these changes addressing accessibility test fails? if so, have you got Dalin's latest changes in this branch to make sure they don't conflict?